### DISCUSSION TITLE
Home Assistant OS 13.2.rc2

### DISCUSSION THREAD

**Discussion by sairon** [home-assistant/operating-system#3624] [Announcements]
:information_source: **This beta release contains an [updated landing page](https://github.com/home-assistant/landingpage/releases/tag/2024.10.0) and a new version of the bundled Supervisor required for its function. For most of the users the changes compared to the previous beta are minimal.**

Other changes since [13.2.rc1](https://github.com/home-assistant/operating-system/releases/tag/13.2.rc1):

## Home Assistant Green

* Linux: Update kernel to 6.6.54 (#3614) @sairon

## Open Virtual Appliance

* Linux: Update kernel to 6.6.54 (#3614) @sairon

## Generic x86-64

* Linux: Update kernel to 6.6.54 (#3614) @sairon

## Hardkernel ODROID

* Linux: Update kernel to 6.6.54 (#3614) @sairon

## ASUS Tinker

* Linux: Update kernel to 6.6.54 (#3614) @sairon

## Khadas VIM Series

* Linux: Update kernel to 6.6.54 (#3614) @sairon

## Generic aarch64

* Linux: Update kernel to 6.6.54 (#3614) @sairon

## Dependencies

* Bump docker/setup-buildx-action from 3.6.1 to 3.7.1 (#3613) @dependabot
* Bump docker/build-push-action from 6.7.0 to 6.9.0 (#3604) @dependabot

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/13.2.rc2'>Home Assistant OS 13.2.rc2</a>.</em>

**Comment by Anto79-ops** [comment]
There are no issues on my NUC 12 (x86-64, bare-metal) with USB SkyConnect and USB Aeotec Z-Stick 7.

System is running normally.
